components/lib/column/Column.vue

```diff
@@ -4,6 +4,18 @@ import BaseColumn from './BaseColumn.vue';
 export default {
     name: 'Column',
     extends: BaseColumn,
+    inject: ['registerColumn', 'unregisterColumn'],
+    mounted() {
+        if (this.registerColumn) {
+            this.$.children = new Proxy(this.$.slots, {});
```

Comment:
We need to construct a proxy to let DataTable components (f.e. TableHeader) access `column.children` to access the slots.